app/views/trainees/deferrals/show.html.erb

```diff
@@ -0,0 +1,32 @@
+<%= render PageTitle::View.new(title: "trainees.deferral.show", has_errors: @deferral.errors.present?) %>
+
+<%= content_for(:breadcrumbs) do %>
+  <%= render GovukComponent::BackLink.new(
+    text: 'Back',
```

Comment:
In John's PR he added a new translation for 'Back' links. For consistency, it would be good to use that here (and in your other 'Back' link on the confirm page.

Reply:
Thanks, fixed this